packages/squiggle-lang/__tests__/ReducerInterface/ReducerInterface_Distribution_test.res

```diff
@@ -0,0 +1,28 @@
+open Jest
+open Reducer_TestHelpers
+
+let makeTest = (str, result) => test(str, () => expectEvalToBe(str, result))
```

Comment:
I think the file would read more naturally if this was `testEval`

Reply:
Good point, will change